__test__/getProfiles.test.ts

```diff
@@ -0,0 +1,19 @@
+import { ThreadsAPI } from '../src/threads-api';
+
+test("UserName Unit test", async () => {
```

Comment:
Can we rename this to be more descriptive?

Reply:
Okay! I'll modify that 😄